src/main/java/io/gatling/mojo/EnterpriseStartMojo.java

```diff
@@ -0,0 +1,124 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gatling.mojo;
+
+import io.gatling.plugin.util.EnterpriseClient;
+import io.gatling.plugin.util.exceptions.EnterpriseClientException;
+import io.gatling.plugin.util.model.RunSummary;
+import io.gatling.plugin.util.model.SimulationAndRunSummary;
+import java.io.File;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugin.MojoFailureException;
+import org.apache.maven.plugins.annotations.Execute;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.apache.maven.plugins.annotations.ResolutionScope;
+
+@Execute(goal = "enterprisePackage")
+@Mojo(name = "enterpriseStart", requiresDependencyResolution = ResolutionScope.TEST)
+public class EnterpriseStartMojo extends AbstractEnterpriseApiMojo {
+
+  /** A name of a Simulation class to run. */
+  @Parameter(property = "gatling.simulationClass")
+  private String simulationClass;
+
+  @Parameter(readonly = true)
+  private String simulationId;
+
+  @Parameter(readonly = true)
+  private Map<String, String> simulationSystemProperties;
+
+  @Override
+  public void execute() throws MojoExecutionException, MojoFailureException {
+    checkPluginPreConditions();
+
+    if (simulationSystemProperties == null) {
+      // @Parameter(defaultValue = ...) only works for properties with a single value
+      simulationSystemProperties = Collections.emptyMap();
+    }
+
+    final File file = shadedArtifactFile();
+    final EnterpriseClient enterpriseClient = initEnterpriseClient();
+
+    try {
+      final RunSummary runSummary;
+      if (simulationId == null) {
+        getLog().info("Creating and starting simulation...");
+        final SimulationAndRunSummary result =
+            enterpriseClient.createAndStartSimulation(
+                mavenProject.getGroupId(),
+                mavenProject.getArtifactId(),
+                simulation(),
+                simulationSystemProperties,
+                file);
+
+        final String simulationId = result.simulation.id.toString();
+        final String simulationName = result.simulation.name;
+        runSummary = result.runSummary;
+        getLog()
+            .info(String.format("Created simulation %s with ID %s", simulationName, simulationId));
+        final String configMsg =
+            "To start again the same simulation, add the 'simulationId' configuration to your pom.xml, e.g.:\n"
+                + "<plugin>\n"
+                + "  <groupId>io.gatling</groupId>\n"
+                + "  <artifactId>gatling-maven-plugin</artifactId>\n"
+                + "  <configuration>\n"
+                + "    <simulationId>"
```

Comment:
We could also show the configuration for packageId, even if it is unlikely to be used.